pkg/types/types.go

```diff
@@ -28,3 +28,8 @@ type ChartVersionRepository struct {
 	Chart         string
 	ChartVersions repo.ChartVersions
 }
+type HelmVerifierRequest struct {
+	ChartUrl string                 `json:"chart_url"`
+	Chartset []string               `json:"chart_set"`
```

Comment:
Please use `PascalCase` instead of `snake_case` for serialized fields.

```suggestion
	ChartUrl string                 `json:"chartUrl"`
	Chartset []string               `json:"chartSet"`
```